src/components/cards/cards.scss

```
@@ -49,12 +49,18 @@
   .logo-row {
     display: flex;
     justify-content: space-between;
+    padding-left: 15px;
+    padding-right: 5px;
 
     .icon {
       color: var(--pf-global--info-color--100);
       margin-right: 3px;
     }
 
+    .left-container {
```

```suggestion
    .hub-left-container {
```

(let's not add new unprefix classnames)